### PR TITLE
[Bug 13180]  Allow GPS access from Android Browser Control

### PR DESCRIPTION
**Pull request by livecodepanos**
Note: You still have to check the Location permissions in the Android Standalone Settings

### CODE REVIEW

File: engine/src/java/com/runrev/android/libraries/LibBrowser.java [hunk 1]
```diff
@@ -321,10 +321,14 @@ public void onHideCustomView()
 					m_custom_view_callback = null;
 				}
 			}
+			public void onGeolocationPermissionsShowPrompt(String origin, GeolocationPermissions.Callback callback) {
+				callback.invoke(origin, true, false);
```
Comment (montegoulding) [inline engine/src/java/com/runrev/android/libraries/LibBrowser.java]: @livecodepanos I'm thinking we should be presenting a dialog here

Comment (livecodepanos) [reply engine/src/java/com/runrev/android/libraries/LibBrowser.java]: Sure, I'll look into adding one

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 5e91c0f

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 5e91c0f074086dfa09501a59086f875b1d70b731

**Comment by livecode-vulcan** [comment]
😎 test success 5e91c0f074086dfa09501a59086f875b1d70b731

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/1313)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1329)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1303)
* try-community-universal-ios-iphoneos11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.2/builds/256)
* try-community-universal-ios-iphonesimulator11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.2/builds/254)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/1307)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1333)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/1313)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1328)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/1312)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1322)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1364)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1303)

**Pull request merged**